### ISSUE TITLE
Add jsoncrack to Open VSX Registry

### ISSUE THREAD

**Issue by neiesc** [AykutSarac/jsoncrack.com#233]
https://open-vsx.org/ is for [VSCodium - Open Source Binaries of VSCode](https://vscodium.com/)

**Comment by AykutSarac** [comment]
Thanks for the idea, I think I'll skip it for now as there are a lot to manage currently.

**Comment by midnight-wonderer** [comment]
Could you consider reopening this?

In case you missed the news
* [The Register: 
Devs sound alarm after Microsoft subtracts C/C++ extension from VS Code forks](https://www.theregister.com/2025/04/24/microsoft_vs_code_subtracts_cc_extension/)
* [YouTube: Microsoft Goes To War With VSCode Forks](https://youtu.be/fz6XJDq8fSI)

I too never considered VSCodium over just some telemetry data Microsoft sent.  
But they are now actively blocking forks from official extensions, despite API compatibility.  
Basically, putting in more work to ensure other people can't use their piece of software.
TBF: they have all the rights to do so.  

But, I switched to VSCodium right away.

I might not be able to convince you to switch, but could you help this side too?  
By handing some power to forks, you will help keep Microsoft at bay which, indirectly, protect the features you love about VSCode.
